examples/deploy/deploy.mjs

```diff
@@ -0,0 +1,34 @@
+import { Policy, t, typegraph } from "@typegraph/sdk/index.js";
+import { DenoRuntime } from "@typegraph/sdk/runtimes/deno.js";
+import { tgDeploy } from "@typegraph/sdk/tg_deploy.js";
```

Comment:
.mjs is required or can we use simple .ts or .mts?

Reply:
No mjs is not required, but I guess it's the minimal setup for a node/deno typegraph without typescript.